dm/tests/shardddl_optimistic/run.sh

```
@@ -461,6 +461,31 @@ function DM_CREATE_DROP_TABLE() {
+function DM_CHANGE_UTF_CHARSET_CASE() {
+	run_sql_source1 "alter table ${shardddl1}.${tb1} character set utf8mb4 collate utf8mb4_bin;"
+	run_sql_source1 "insert into ${shardddl1}.${tb1} values(1, 'a');"
+	run_sql_source1 "alter table ${shardddl1}.${tb2} character set utf8mb4 collate utf8mb4_bin;"
+	run_sql_source1 "insert into ${shardddl1}.${tb2} values(2, 'b');"
+	run_sql_source2 "alter table ${shardddl1}.${tb1} character set utf8mb4 collate utf8mb4_bin;"
+	run_sql_source2 "insert into ${shardddl1}.${tb1} values(3, 'c');"
+	run_sql_source2 "alter table ${shardddl1}.${tb2} character set utf8mb4 collate utf8mb4_bin;"
+	run_sql_source2 "insert into ${shardddl1}.${tb2} values(4, 'd');"
+
+	run_sql_tidb_with_retry "select count(1) from ${shardddl}.${tb};" "count(1): 4"
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)